client/src/components/NavBar.jsx

```diff
@@ -0,0 +1,41 @@
+import { NavLink } from 'react-router-dom';
+import { ReactComponent as Home } from '../icons/home.svg';
+import styled from 'styled-components';
+
+const NavBar = () => {
+  return (
+    <NavWrapper>
```

Comment:
You do have a navBar with just one Link? Are you sure you need the navbar yet? 
Usually we do not implement elements that we dont need :octocat: